### DISCUSSION TITLE
Documentation for Crop CLI

### DISCUSSION THREAD

**Discussion by DSLituiev** [libvips/libvips#2088] [Ideas]
**Is your feature request related to a problem? Please describe.**
Documentation for `vips crop`, hopefully [here](https://libvips.github.io/libvips/API/current/using-cli.html)

I saw `vips crop` command in this [SO post](
https://stackoverflow.com/a/48376269/894763), but google finds only `smartcrop`  command elsewhere. 

I would like to understand :
- what is the order of location parameters (offset-x, offset-y, width or end-x, height or end-y)
- how to make it preserve the original format. I have a pyramidal TIFF64. When I convert it, I get TIFF.

**Describe the solution you'd like**
a documentation page

**Comment by jcupitt** [comment]
Hi @DSLituiev,

It's documented here:

https://libvips.github.io/libvips/API/current/libvips-conversion.html#vips-crop

**Comment by jcupitt** [comment]
On you second point, libvips does not preserve formats. All images are loaded as simple, large untyped arrays, and all formats are recreated again on save. Loaders can attach hints about save properties, but that's all.

For example, you can enter:

```
vips crop my-huge-pyramid.tiff[page=2] out.tif[pyramid,tile,compression=jpeg,Q=80,bigtiff] 100 100 10000 10000
```

Now it'll load page 2 (the x4 reduction level) from your original pyramid, crop out a 10,000 by 10,000 pixel area, and write as a new tiff pyramid.
